### PR TITLE
EPMRPP-83282 || Update profile page layout

### PR DESCRIPTION
**Pull request by Bam6ycha**

### CODE REVIEW

File: app/src/pages/inside/profilePage/profilePage.jsx [hunk 1]
```diff
@@ -35,32 +39,82 @@ const messages = defineMessages({
...
+    id: 'ProfilePage.projectAssignmentTab',
+    defaultMessage: 'PROJECT ASSIGNMENT',
+  },
+  profilePageProjectApiKeysTab: {
+    id: 'ProfilePage.apiKeys',
+    defaultMessage: 'API KEYS',
+  },
+  profilePageConfigurationExamplesTab: {
+    id: 'ProfilePage.configurationExamples',
+    defaultMessage: 'CONFIGURATION EXAMPLES',
```
Comment (AmsterGet) [inline app/src/pages/inside/profilePage/profilePage.jsx:52]: Please use lower case.

File: app/src/pages/inside/profilePage/profilePage.jsx [hunk 2]
```diff
@@ -35,32 +39,82 @@ const messages = defineMessages({
...
+  },
+});
+
+const getDefaultLinkConfig = (profileRoute) => ({
```
Comment (AmsterGet) [inline app/src/pages/inside/profilePage/profilePage.jsx:56]: Why it is `default` as it is customized for every particular route every time?
Lets name it as `getProfilePageLink`.

File: app/src/pages/inside/profilePage/profilePage.jsx [hunk 3]
```diff
@@ -35,32 +39,82 @@ const messages = defineMessages({
...
-            <AssignedProjectsBlock />
-            <LocalizationBlock />
-          </div>
-          <div className={cx('column')}>
-            <AccessTokenBlock />
-            <ConfigExamplesBlock />
+      <div className={cx('container')}>
+        <section className={cx('content-wrapper')}>
+          <div>
+            <article className={cx('section-wrapper')}>
```
Comment (AmsterGet) [inline app/src/pages/inside/profilePage/profilePage.jsx:106]: I'm not sure that `article` tag relevant here as we don't store block of text information here.

File: app/src/pages/inside/profilePage/profilePage.jsx [hunk 4]
```diff
@@ -35,32 +39,82 @@ const messages = defineMessages({
...
-            <PersonalInfoBlock />
-            <AssignedProjectsBlock />
-            <LocalizationBlock />
-          </div>
-          <div className={cx('column')}>
-            <AccessTokenBlock />
-            <ConfigExamplesBlock />
+      <div className={cx('container')}>
+        <section className={cx('content-wrapper')}>
+          <div>
```
Comment (AmsterGet) [inline app/src/pages/inside/profilePage/profilePage.jsx:105]: Do we need an empty `div` here?

File: app/src/routes/routesMap.js [hunk 5]
```diff
@@ -104,7 +104,7 @@ const routesMap = {
   [NOT_FOUND]: '/notfound',
 
   ADMINISTRATE_PAGE: redirectRoute('/administrate', () => ({ type: PROJECTS_PAGE })),
-  USER_PROFILE_PAGE: '/user-profile',
+  USER_PROFILE_PAGE: '/user-profile/:profileRoute*',
```
Comment (AmsterGet) [inline app/src/routes/routesMap.js:107]: As we have predefined routes and don't support any dynamic, let's declare it here.
Refer [SERVER_SETTINGS_TAB_PAGE](https://github.com/reportportal/service-ui/pull/3465/files#diff-256b4742683e2623bba9b071bc6fc31c9fcf42a98e610945b970b50acde3b737R132) with its sub routes.

File: app/src/pages/inside/profilePage/constants.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+export const API_KEYS_ROUTE = 'apiKeys';
+export const PROJECT_ASSIGNMENT_ROUTE = 'assignedProjects';
+export const CONFIG_EXAMPLES_ROUTE = 'configExamples';
```
Comment (AmsterGet) [inline app/src/pages/inside/profilePage/constants.js:3]: Let's move these constants to `common/constants` as we store all sub routes there.
And don't forget to add EPAM signature.

File: app/src/controllers/pages/selectors.js [hunk 7]
```diff
@@ -54,6 +54,8 @@ export const pluginsTabSelector = (state) => payloadSelector(state).pluginsTab;
 export const pluginPageSelector = (state) => payloadSelector(state).pluginPage;
 export const pluginRouteSelector = (state) => payloadSelector(state).pluginRoute;
 
+export const adminProfileRouteSelector = (state) => payloadSelector(state).profileRoute;
```
Comment (AmsterGet) [inline app/src/controllers/pages/selectors.js:57]: As it is not related only for admin, I suggest to rename it to `userProfileRouteSelector`.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/pages/inside/profilePage/profilePage.jsx (hunk 4 above)
Comment (Bam6ycha) [reply app/src/pages/inside/profilePage/profilePage.jsx:105]: Yes. We need it to join navbar and user profile content. And handle it like one flex-item inside `content-wrapper` parent.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**